client/elements/addons/sc-top-sheet-views.js

```diff
@@ -348,28 +348,27 @@ export class SCTopSheetViews extends LitLocalized(LitElement) {
     document.removeEventListener('keydown', this._handleKeydown);
   }
 
+  _handleNKeydown() {
+    let idx = this.noteDisplayTypeArray.findIndex(item => item.displayType === this.selectedNoteDisplayType);
+    if (idx >= 0) {
+      idx = (idx + 1) % this.noteDisplayTypeArray.length;
+      this.changeNoteDisplayType(this.noteDisplayTypeArray[idx].displayType);
+    }
+  }
+
+  _handleVKeydown() {
+    let idx = this.textViewArray.findIndex(item => item.textView === this.selectedTextView);
+    if (idx >= 0) {
+      idx = (idx + 1) % this.textViewArray.length;
+      this._changeTextView(this.textViewArray[idx].textView);
+    }
+  }
+
   _handleKeydown(event) {
     if(ignorableKeydownEvent(event)) return;
-    let idx = -1;
-    switch (event.key) {
-      case 'n':
-      case 'N':
-        idx = this.noteDisplayTypeArray.findIndex(item => item.displayType === this.selectedNoteDisplayType);
-        if (idx >= 0) {
-          idx = (idx + 1) % this.noteDisplayTypeArray.length;
-          this.changeNoteDisplayType(this.noteDisplayTypeArray[idx].displayType);
-        }
-        break;
-      case 'v':
-      case 'V':
-        idx = this.textViewArray.findIndex(item => item.textView === this.selectedTextView);
-        if (idx >= 0) {
-          idx = (idx + 1) % this.textViewArray.length;
-          this.#setTextBilaraPageIsTextOptionsMismatchSavedSettingsState();
-          this._changeTextView(this.textViewArray[idx].textView);
-        }
-        break;
-    }
+    if(!(event.key in this._keydownHandlers)) return;
+    this.#setTextBilaraPageIsTextOptionsMismatchSavedSettingsState();
```

Comment:
@thesunshade - with the `_handleKeydown` function written like this, now I don't have to remember to call this function every time I add a new key binding.